theme/scripts/fadeIn.ts

```diff
@@ -0,0 +1,72 @@
+import isInViewport from './utils/isInViewport';
+
+export default (function() {
+  const FadeIn = {
+    SELECTORS: {
+      homeGridItem: "[data-home-grid-item]",
+      collectionGridItem: "[data-collection-grid-item]",
+    },
+    ATTRIBUTES: {
+      fadeIn: "data-fade-in",
+    },
+
+    init() {
+      //Fade in activated by scroll
+      const homeGridItems = [].slice.call(
+        document.querySelectorAll(FadeIn.SELECTORS.homeGridItem)
+      )
+
+      //Fade in activated by page load, then by scroll
+      const collectionGridItems = [].slice.call(
+        document.querySelectorAll(FadeIn.SELECTORS.collectionGridItem)
+      )
```

Comment:
Home Grid Items are activated by scroll (since they are not at the top of the page), but Collection pages need to have the animation activated on page load... this works but wasn't sure if this was the best way to do it!

Reply:
First of all, this is looking so fire. So great job. 

I think a better way to approach this might be to treat them the same, and just call activate when the document loads. That way, on load you fade in any images that are considered `in view` and then can handle fading in images on scroll as that event is triggered.